asv/benchmarks/setup/bench_model.py

```
@@ -47,13 +47,44 @@ def time_initialize_model(self, robot, num_envs):
+        if robot == "h1":
+            # use more samples for H1 to reduce variance
+            self.number = 30
+
+        wp.init()
+        builder = Example.create_model_builder(robot, num_envs, randomize=True, seed=123)
+
+        # finalize model
+        self._model = builder.finalize()
+
+    @skip_benchmark_if(wp.get_cuda_device_count() == 0)
+    def time_initialize_solver(self, robot, num_envs):
```

Ruff (0.12.2)

Unused method argument: `num_envs`

(ARG002)